### PR TITLE
Added support for building docker image

### PR DESCRIPTION
**Pull request by ICoxCH**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (ICoxCH) [inline .dockerignore:1]: Do not copy anything covered by this file into the docker image.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:14
```
Comment (ICoxCH) [inline Dockerfile:1]: Start with a minimised linux distribution with Node 14 installed

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Create app directory
+WORKDIR /usr/src/app
+
+COPY package*.json ./
```
Comment (ICoxCH) [inline Dockerfile:6]: Move both package.json and package-lock.json into the image

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN npm install
```
Comment (ICoxCH) [inline Dockerfile:8]: download dependencies for application

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+COPY . .
```
Comment (ICoxCH) [inline Dockerfile:10]: copy everything not covered by .dockerignore into the image

File: Dockerfile [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+EXPOSE 3000
```
Comment (ICoxCH) [inline Dockerfile:12]: open port 3000 on image this lines up with port that the application listens on. Allowing the app to be hit by the outside world

File: Dockerfile [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+CMD [ "node", "./bin/www" ]
```
Comment (ICoxCH) [inline Dockerfile:14]: when the image is spun up, the command that is ran `node ./bin/www`

**Review by amyatt1CH — CHANGES_REQUESTED** [review]
All looks fine - only thing I'd mention is updating the README to show usage with docker. e.g.
`run docker build -t onboarding-portal-webapp .` then run `docker run -t -i -p 3000:3000 onboarding-portal-webapp` (they don't have to be these exact commands or arguments, just an example)

**Review by amyatt1CH — APPROVED** [review]
👍

**Pull request merged**